### PR TITLE
feat: add api key to bff requests

### PR DESCRIPTION
**Pull request by florian-hoenicke**
At the moment, it is working to use API keys and modify them when talking to the gateway. In this pr sending the api_key requests to the bff will be supported.

### CODE REVIEW

File: now/admin/search.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import requests
+from docarray import Document
+
+from deployment.bff.app.v1.routers.helper import jina_client_post
+
+API_KEY = 'my_key'
+
+url = f"https://nowrun.jina.ai/api/v1/text-to-image/search"
```
Comment (makram93) [inline now/admin/search.py:8]: Is this file meant to be a local testing file?

Comment (florian-hoenicke) [reply now/admin/search.py:8]: yes - just for us

**Review by makram93 — APPROVED** [review]
LGTM!

**Review by joschkabraun — APPROVED** [review]
LGTM! Nice refactor

**Pull request merged**